dts/bindings/sensor/ti,ina226.yaml

```
@@ -0,0 +1,73 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    TI INA226 Bidirectional Current and Power Monitor.
+    The <zephyr/dt-bindings/sensor/ina226.h> file should be included in the
+    DeviceTree and it provides macros that can be used for initializing the
```

missed this small one before - should there be another round of updates: 
```suggestion
    DeviceTree as it provides macros that can be used for initializing the
```